plots/mock/README.md

```diff
@@ -0,0 +1,25 @@
+# Mock plots
+
+-Is there a relationship between predator size (mass and length) and prey size (mass and length)?
+
+Caption: 
+Line graph representing a positive correlation between predator size (mass /length) and prey size (mass / length)
+
+-Does the same species of predator in different geographic locations exhibit the same feeding pattern? Are they eating the same sizes of prey in different locations?
+
+Caption:
+1. Bar plot representing the average prey size of specific predators in specific geographic locations
+2. Box plot representing the range of prey size of specific predators in specific geographic locations 
+
+-Does geography (latitude, longitude, geographic location, specific habitat) influence predator and prey sizes (mass or length ratios)?
+
+Caption:
+Bar plot representing predator:prey mass ratios in different geographic locations 
+
+-Do environmental factors (depth, mean annual temperature, mean primary productivity, specific habitat) influence feeding interactions or predator and prey size relationships (mass, length, fork length)?
+
+Caption: 
+Line graph depicting how a range of an environmental factors affect predator size 
```

Comment:
I'd just re-phrase this as something like: The variation of predator size across different levels of a given environmental factor